pipeline/scraper/retriever.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+//const config = require('/etc/xray/config.json'); //See example_config.json
+const gplay = require('google-play-scraper');
+const logger = require('./logger.js');
+const _ = require('lodash');
+var database = require('./db.js');
+var db = new database('explorer');
+
+const region = 'us';
+
+/**
+ * Inserts app data into the db using db.js
+ * @param {*The app data json that is to be inserted into the databae.} app_data 
+ */
+function insertAppData(app_data) {
+
+    //Checking version data - correct version to update date
+    if (!app_data.version || app_data.version === 'Varies with device') {
```

Comment:
Agreed. Should revisit later after Ret' Exp' and Down' are all up and running in an acceptable state.